generated/nirfmxnr/nirfmxnr_compilation_test.cpp

```diff
@@ -667,6 +667,11 @@ int32 ModAccFetchPUSCHPTRSConstellationTrace(niRFmxInstrHandle instrumentHandle,
   return RFmxNR_ModAccFetchPUSCHPTRSConstellationTrace(instrumentHandle, selectorString, timeout, puschptrsConstellation, arraySize, actualArraySize);
 }
 
+int32 ModAccFetchPUSCHPhaseOffsetTrace(niRFmxInstrHandle instrumentHandle, char selectorString[], float64 timeout, float64* x0, float64* dx, float32 puschPhaseOffset[], int32 arraySize, int32* actualArraySize)
+{
+  return RFmxNR_ModAccFetchPUSCHPhaseOffsetTrace(instrumentHandle, selectorString, timeout, x0, dx, puschPhaseOffset, arraySize, actualArraySize);
```

Comment:
I think you'll need to bump the version of niRFmxNR.h or it will fail the build because of this compilation "test". You can update it here to match what you're updating it to in scrapigen: https://github.com/ni/grpc-device/blob/main/imports/include/niRFmxNR.h